app/pages/organisation/addproxy/controller.js

```diff
@@ -0,0 +1,106 @@
+import { getData, postData, ErrorContext } from 'buying-catalogue-library';
+import { logger } from '../../../logger';
+import { getEndpoint } from '../../../endpoints';
+
+export const getUnrelatedOrganisations = async ({ organisationId, accessToken }) => {
+  try {
+    const endpoint = getEndpoint({ endpointLocator: 'getUnrelatedOrgs', options: { organisationId } });
+    const unrelatedOrgs = await getData({ endpoint, accessToken, logger });
+    logger.info(`Unrelated organisations for ${organisationId} returned`);
+    return unrelatedOrgs;
+  } catch (err) {
+    const errorString = `Error finding unrelated organisation data for id ${organisationId}`;
+    logger.error(errorString);
+    throw new ErrorContext({
+      status: err.response.status,
+      description: errorString,
+    });
+  }
+};
+
```

Comment:
This 'ErrorContext' will take you to an error page with the appropriate error message (errorString) which I think is a lot nicer than 'something went wrong'.

Reply:
I'm all for something better than the 'something went wrong' message. My only concern with this approach is the amount of boiler plate code introduced. However, let's get this in as is. It's probably not even worth looking into whether we can make this more DRY (unless it's straight-forward/quick) due to the current architectural work.

Reply:
It's not actually significantly more than what is done generally, I am just throwing an ErrorContext instead of an empty error.